[2025-03-14T10:46:27.462+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-14T10:46:27.521+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: A_dag_with_taskflow_api_v01.get_name scheduled__2025-03-12T00:00:00+00:00 [queued]>
[2025-03-14T10:46:27.538+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: A_dag_with_taskflow_api_v01.get_name scheduled__2025-03-12T00:00:00+00:00 [queued]>
[2025-03-14T10:46:27.539+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 5
[2025-03-14T10:46:27.579+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): get_name> on 2025-03-12 00:00:00+00:00
[2025-03-14T10:46:27.602+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=248) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-14T10:46:27.610+0000] {standard_task_runner.py:72} INFO - Started process 274 to run task
[2025-03-14T10:46:27.594+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'A_dag_with_taskflow_api_v01', 'get_name', 'scheduled__2025-03-12T00:00:00+00:00', '--job-id', '111', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_taskflow_api.py', '--cfg-path', '/tmp/tmp757i06wg']
[2025-03-14T10:46:27.615+0000] {standard_task_runner.py:105} INFO - Job 111: Subtask get_name
[2025-03-14T10:46:27.940+0000] {task_command.py:467} INFO - Running <TaskInstance: A_dag_with_taskflow_api_v01.get_name scheduled__2025-03-12T00:00:00+00:00 [running]> on host 857b64da211c
[2025-03-14T10:46:28.280+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='A_dag_with_taskflow_api_v01' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2025-03-12T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-12T00:00:00+00:00'
[2025-03-14T10:46:28.282+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-14T10:46:28.292+0000] {python.py:240} INFO - Done. Returned value was: Christophe
[2025-03-14T10:46:28.351+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-14T10:46:28.352+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=A_dag_with_taskflow_api_v01, task_id=get_name, run_id=scheduled__2025-03-12T00:00:00+00:00, execution_date=20250312T000000, start_date=20250314T104627, end_date=20250314T104628
[2025-03-14T10:46:28.435+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-14T10:46:28.466+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
